static-site/src/components/Cards/coreCards.js

```diff
@@ -1,12 +1,12 @@
 const coreCards = [
   {
     img: "mpox.png",
-    url: "/mpox",
+    url: "/mpox/clade-IIb",
     title: "Mpox"
   },
   {
     img: "seasonalinfluenza.png",
-    url: "/flu/seasonal",
+    url: "/seasonal-flu/h3n2/ha/2y",
```

Comment:
Do we want to specify the exact URLs here or should they go through the redirect so that they always to go to the default dataset that is specified in the `manifest_core.json`?

Reply:
I changed to full paths to avoid what I felt was an unnecessary redirect cycle. The upside is faster loading pages, although the actual improvement may be tiny. The downside is decoupling the manifest seasonal-flu default from the splash page tile dataset. Although maybe that's an upside not a downside.

Reply:
> Although maybe that's an upside not a downside.

True! Now we know for sure which builds these cards link to without having to go through the manifest!

Reply:
👍  I'll update the commit comment when I fixup the PR (post review)

Reply:
Commit reworded